testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/SecurityCommandsTestCase.java

```diff
@@ -77,11 +97,266 @@
+            Assert.assertTrue(cli.pushLineAndWaitForResults("n", null));
+            Assert.assertTrue(cli.getOutput().contains("Ignoring, command not executed. You need to accept the TOS to create account and obtain certificates."));
+        } catch (Throwable ex) {
+            throw new Exception(cli.getOutput(), ex);
+        } finally {
+            cli.destroyProcess();
+        }
+    }
+
+    @Test
+    public void testEnableLetsEncryptSSLInteractiveUseFakeCaAccount() throws Exception {
+        CliProcessWrapper cli = new CliProcessWrapper().
+                addJavaOption("-Duser.home=" + temporaryUserHome.getRoot().toPath().toString()).
+                addCliArgument("--controller=remote+http://"
+                        + TestSuiteEnvironment.getServerAddress() + ":"
+                        + TestSuiteEnvironment.getServerPort()).
+                addCliArgument("--connect");
+
+        try {
+            cli.executeInteractive();
+            cli.clearOutput();
+
+            Path jbossStandaloneConfig = Paths.get(TestSuiteEnvironment.getJBossHome() +"/standalone/configuration");
+            Path KSFile = jbossStandaloneConfig.resolve(KEYSTORE_FILE);
+
+            //Copy the pre-generated key stores to the server config
+            Files.copy(AcmeMockServerBuilder.class.getResourceAsStream(ACCOUNTS_KEYSTORE_FILE_NAME),
+                    jbossStandaloneConfig.resolve(ACCOUNTS_KEYSTORE_FILE_NAME),
+                    java.nio.file.StandardCopyOption.REPLACE_EXISTING);
+            Files.copy(AcmeMockServerBuilder.class.getResourceAsStream(KEYSTORE_FILE), KSFile, java.nio.file.StandardCopyOption.REPLACE_EXISTING);
+            //Check that the key-store does not contain server certificate
+            assertFalse(loadKeyStore(KSFile.toString(), KEYSTORE_PASSWORD).containsAlias(certificateAlias));
+
+            Assert.assertTrue(cli.pushLineAndWaitForResults("security enable-ssl-management --interactive --no-reload"
+                            + " --lets-encrypt"
+                            + " --ca-account=FakeCaAccount", "Key-store file name (default management.keystore):" ));
+            Assert.assertTrue(cli.pushLineAndWaitForResults(KEYSTORE_FILE, "Password (blank generated)"));
+            Assert.assertTrue(cli.pushLineAndWaitForResults(KEYSTORE_PASSWORD, "Your domain name(s) (must be accessible by the Let's Encrypt server at 80 & 443 ports) [example.com,second.example.com]"));
+            Assert.assertTrue(cli.pushLineAndWaitForResults("inlneseppwkfwew.com", "Alias (blank generated):"));
+            Assert.assertTrue(cli.pushLineAndWaitForResults(certificateAlias, "Enable SSL Mutual Authentication"));
+            Assert.assertTrue(cli.pushLineAndWaitForResults("n", "Do you confirm"));
+            Assert.assertTrue(cli.pushLineAndWaitForResults("y", null));
+            Assert.assertTrue(cli.getOutput().contains("FakeCaAccount not found"));
```

Comment:
This is not the problem for this pull request.
But while I review the test with fake Ca Account, I discover in the server log word service twice.
Does anyone have an idea why is word service in the exception message written twice?


````
6:39:03,285 ERROR [org.jboss.as.controller.management-operation] (management-handler-thread - 1) WFLYCTL0013: Operation ("obtain-certificate") failed - address: ([
    ("subsystem" => "elytron"),
    ("key-store" => "key-store-f7d7a87a-9df1-4591-8a1d-5cd2d775a9f0")
]): org.jboss.msc.service.ServiceNotFoundException: Service service org.wildfly.security.certificate-authority-account.FakeCaAccount not found
	at org.jboss.msc.service.ServiceContainerImpl.getRequiredService(ServiceContainerImpl.java:652)
	at org.jboss.as.controller.OperationContextImpl$OperationContextServiceRegistry.getRequiredService(OperationContextImpl.java:2282)
	at org.wildfly.extension.elytron.ElytronExtension.getRequiredService(ElytronExtension.java:138)
	at org.wildfly.extension.elytron.AdvancedModifiableKeyStoreDecorator.getAcmeAccount(AdvancedModifiableKeyStoreDecorator.java:1015)
	at org.wildfly.extension.elytron.AdvancedModifiableKeyStoreDecorator.access$100(AdvancedModifiableKeyStoreDecorator.java:104)
	at org.wildfly.extension.elytron.AdvancedModifiableKeyStoreDecorator$ObtainCertificateHandler.executeRuntimeStep(AdvancedModifiableKeyStoreDecorator.java:786)
	at org.jboss.as.controller.AbstractRuntimeOnlyHandler$1.execute(AbstractRuntimeOnlyHandler.java:59)
	at org.jboss.as.controller.AbstractOperationContext.executeStep(AbstractOperationContext.java:999)
	at org.jboss.as.controller.AbstractOperationContext.processStages(AbstractOperationContext.java:743)
	at org.jboss.as.controller.AbstractOperationContext.executeOperation(AbstractOperationContext.java:467)
	at org.jboss.as.controller.OperationContextImpl.executeOperation(OperationContextImpl.java:1412)
	at org.jboss.as.controller.ModelControllerImpl.internalExecute(ModelControllerImpl.java:423)
	at org.jboss.as.controller.ModelControllerImpl.lambda$execute$1(ModelControllerImpl.java:243)
	at org.wildfly.security.auth.server.SecurityIdentity.runAs(SecurityIdentity.java:289)
	at org.wildfly.security.auth.server.SecurityIdentity.runAs(SecurityIdentity.java:255)
	at org.jboss.as.controller.ModelControllerImpl.execute(ModelControllerImpl.java:243)
	at org.jboss.as.controller.remote.ModelControllerClientOperationHandler$ExecuteRequestHandler.doExecute(ModelControllerClientOperationHandler.java:240)
	at org.jboss.as.controller.remote.ModelControllerClientOperationHandler$ExecuteRequestHandler.access$400(ModelControllerClientOperationHandler.java:138)
	at org.jboss.as.controller.remote.ModelControllerClientOperationHandler$ExecuteRequestHandler$1$1.run(ModelControllerClientOperationHandler.java:162)
	at org.jboss.as.controller.remote.ModelControllerClientOperationHandler$ExecuteRequestHandler$1$1.run(ModelControllerClientOperationHandler.java:158)
	at org.wildfly.security.auth.server.SecurityIdentity.runAs(SecurityIdentity.java:313)
	at org.wildfly.security.auth.server.SecurityIdentity.runAs(SecurityIdentity.java:270)
	at org.jboss.as.controller.AccessAuditContext.doAs(AccessAuditContext.java:254)
	at org.jboss.as.controller.AccessAuditContext.doAs(AccessAuditContext.java:225)
	at org.jboss.as.controller.remote.ModelControllerClientOperationHandler$ExecuteRequestHandler$1.execute(ModelControllerClientOperationHandler.java:158)
	at org.jboss.as.protocol.mgmt.ManagementRequestContextImpl$1.doExecute(ManagementRequestContextImpl.java:70)
	at org.jboss.as.protocol.mgmt.ManagementRequestContextImpl$AsyncTaskRunner.run(ManagementRequestContextImpl.java:160)
	at org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
	at org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:1982)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1486)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1377)
	at java.lang.Thread.run(Thread.java:748)
	at org.jboss.threads.JBossThread.run(JBossThread.java:485)

````

Reply:
I created Jira issue for this bug [WFCORE-4515](https://issues.jboss.org/browse/WFCORE-4515)